x-pack/plugins/endpoint/public/applications/endpoint/store/policy_details/selectors.ts

```diff
@@ -27,3 +28,28 @@ export const selectPolicyIdFromParams: (state: PolicyDetailsState) => string = c
     return '';
   }
 );
+
+export const selectPolicyConfig = (state: Immutable<PolicyDetailsState>) => state.policyConfig;
```

Comment:
can I ask that you try to write a doc comment for anything exported?

Reply:
doc comment as in /** stuff here */ right? about what it does